app/helpers/physical_ticket_helper.rb

```diff
@@ -0,0 +1,2 @@
+module PhysicalTicketHelper
```

Comment:
I guess there are aome classes that were automatically generated, but if by now they are not use, like it is the case with this helper, I would better remove them 🤔